test/development/app-dir/ssr-only-error/ssr-only-error.test.ts

```diff
@@ -53,9 +53,14 @@ describe('ssr-only-error', () => {
     const text = await browser.elementByCss('body').text()
     expect(text).toBe('404\nThis page could not be found.')
 
+    // Assert if there's no console.error related to the notFound error
+    // Since there's already one error log containing "404" from browser:
+    // "Failed to load resource: the server responded with a status of 404 (Not Found)"
+    // We assert there's no error text in the error log
     expect(await browser.log()).not.toContainEqual(
       expect.objectContaining({
         source: 'error',
+        message: expect.stringContaining('NEXT_HTTP_ERROR_FALLBACK;404'),
```

Comment:
If we ever change the message, this assertion passes when it really shouldn't. That's why positive assertions are much more robust.

Reply:
How about now? asserting the array only contains one error log that contains a specific text